shortcut_composer/api_krita/enums/action.py

```diff
@@ -0,0 +1,581 @@
+# SPDX-FileCopyrightText: © 2022 Wojciech Trybus <wojtryb@gmail.com>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from krita import Krita as Api
+from enum import Enum
+
+from PyQt5.QtGui import QIcon
+
+
+class Action(Enum):
+    """
+    Contains all known actions of Krita.
+
+    Example usage: `Tool.FREEHAND_BRUSH`
+
+    Available actions:
+    - `TBA`
```

Comment:
Tego nie róbmy - przepisywanie wartości Enuma do docstringa wynikało z tego, że pierwotnie myślałem, że użytkownicy będą sami sobie edytować plik actions.py (ale potem stwierdziłem, że będzie to zniechęcające). Planuje to wyrzucić w pozostałych enumach, bo powiela informacje.

Reply:
Cieszy mnie to niezmiernie hehe :D